ci/release-validate-deps.py

```diff
@@ -0,0 +1,27 @@
+import os
+import sys
+
+if __name__ == '__main__':
+    dependencies_bzl = os.path.join('dependencies', 'graknlabs', 'dependencies.bzl')
+    snapshot_dependencies = []
+    with open(os.path.join(os.getenv("BUILD_WORKSPACE_DIRECTORY"), dependencies_bzl)) as dependencies_bzl_:
+        dependencies_bzl__ = dependencies_bzl_.read().splitlines()
+        for arg in sys.argv[1:]:
+            dependency = filter(lambda line: line.endswith('@' + arg), dependencies_bzl__)
+            if len(dependency) == 1:
```

Comment:
@lolski running this with Python 3 would cause `TypeError: object of type 'filter' has no len()` — pass it to `list()` first

Reply:
Resolved in ad7e6c531b89c5711e31d4ab3e64b650463e56ad